src/renderer/Widget/NotifyWidget/renderer.tsx

```diff
@@ -0,0 +1,84 @@
+import React, {useState, useEffect} from 'react';
+import {getNotifyEvent} from './service';
+import './style.css';
+
+const Notify = ({defaultView, isPreview}: {
+  defaultView?: string;
+  isPreview?: boolean;
+}) => {
+  const initNotifications: any[] = [
+    {
+      url: '',
+      title: 'Sample Notification',
+      body: 'This is body',
+      isRead: false,
+      createdAt: '2017-09-26T06:25:18Z',
+      creator: {
+        name: 'Admin',
+        code: 'Admin'
+      }
+    }
+  ];
+  const [notifications, setNotifications] = useState(initNotifications);
+
+  useEffect(() => {
+    if (!isPreview) {
```

Comment:
The `renderer.tsx` file is used to deploy to kintone, no need of state `isPreview`